src/System.Runtime.Analyzers/CSharp/CSharpInitializeStaticFieldsInline.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Diagnostics;
+using System.Threading;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace System.Runtime.Analyzers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public sealed class CSharpInitializeStaticFieldsInlineAnalyzer : InitializeStaticFieldsInlineAnalyzer<SyntaxKind>
+    {
+        protected override SyntaxKind AssignmentNodeKind => SyntaxKind.SimpleAssignmentExpression;
```

Comment:
What about other kinds of assignment like:
- a, b = 10
- a += 10 (and all the other compound assignments)
- Foo(ref\out a)